docs/contributor_guide/seed_scenarios.rst

```diff
@@ -0,0 +1,85 @@
+
+Seed ideas for scenarios:
+"""""""""""""""""""""""""
+These scenarios are seed ideas for thinking through ways to approach fairness questions, and considering what may be similar or different in your own problem context.  They may be starting points for coming up with new ideas for example notebooks, or some may require other approaches that are out of scope for *fairlearn*.
+
+
+Identifying potential tax fraud
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You're a member of an analytics team in a European country, and brought in to consult about a project that has already started to scale the deployment of models for predicting which tax returns may require further investigation for fraud.  The team has used a model trained in other jurisdictions by a large predictive analytics supplier, and hopes that they can leverage this at a lower cost that would be required to invest in the capability in-house.
+`Veale et al. (2018) <https://arxiv.org/pdf/1802.01029.pdf>`_
+
+Debit card fraud investigation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You're a data scientist at a Dutch financial services company, and your manager asks you to join an existing team.  This team has deploy a model trained on historical transaction data and now new debit transaction data is arriving.  For each new transaction, the model predicts whether it is potentially fraudulent and then will trigger an alert and inspection by human analysts.  The output that matters for the company is the final decision by the human analyst of whether to block the transaction, allow it but flag for further investigation by anothe team, or flag the transaction as normal.  False negatives mean clients can't get their money back (eg, in a phishing scheme), while false positives may overwhelm the team of human analysts or disrupt clients making legitimate purchases.
+`Weerts et al. (2019) <https://arxiv.org/abs/1907.03334>`_
+
+Measuring brand sentiment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You're a member of a team trying to measure brand sentiment from online comments and reviews.  The team hopes to use an existing language model, a third-party service for flagging abusive comments, and then train a more targeted sentiment classifier for your brand on top.
```

Comment:
I'm not clear on the harmed party here. Usually with language models, the issue is that speakers of a particular dialect get poorer QoS. In this context, presumably poorer QoS means that those subgroups will be targeted less in future marketing campaigns (be sent fewer coupons etc.)?

Reply:
I think you're right, Richard. 

This doesn't just apply to brands either (although having a specific context is certainly important!). Let's say you want to target people who will respond to a job ad and then send it to some groups much more than others based on the perceived interest.

Reply:
@riedgar-ms These seem like great questions to be thinking through!  The point of this document is to enable the team to have discussions like you've raised here, so I hope you'll bring these up Thursday! :)

@romanlutz Let's avoid the "portability trap" here :)  We can't discuss **Real harms** as in #490 if we abstract the deployment context away.